### ISSUE TITLE
Could not read from file* because it is not a file

### ISSUE THREAD

**Issue by DaveJarvis** [bedatadriven/renjin#506]
# Replicate

1. Create a script named `file.R`.
1. Create a symbolic link to `file.R` named `link.R`.
1. Call `source( 'link.R' );`.

# Expected

The contents of the symbolic are read successfully.

# Actual

Error message:

> Could not read from file (link.R) because it is not a file

This differs from R's behaviour, which treats symbolic links as regular files:

``` bash
$ ls -la
... possessive.R -> /home/username/dev/R/possessive.R
$ R
> source('possessive.R');
> pos('Renjin');
[1] "Renjin's"
```

**Comment by perNyfelt** [comment]
I did exactly as described:

1. create a script named file.R, content is `print("Hello there")
2. Create a symbolic link to file.R named link.R: `ln -s file.R link.R`
3. Call `source( 'link.R' )`

```
Renjin 3.5-beta76
Copyright (C) 2019 The R Foundation for Statistical Computing
Copyright (C) 2019 BeDataDriven
Falling back to pure JVM BLAS libraries.
> source('link.R')
[1] "Hello there"
> 
```
This was on Linux Mint 19.3. So does not seem to be a problem with Renjin as far as I can tell.

**Comment by DaveJarvis** [comment]
Could it be a version difference? I tried it as well:

``` bash
Renjin 3.5-beta76
Copyright (C) 2019 The R Foundation for Statistical Computing
Copyright (C) 2019 BeDataDriven
Falling back to pure JVM BLAS libraries.
> setwd('/home/username/dev/writing/story/r')
> source('possessive.R')
> pos('Renjin')
[1] "Renjin's"
```

But, when running within an integrated Java desktop application using  `"org.renjin:renjin-script-engine:0.9.2726"`:

![Renjin Error](https://user-images.githubusercontent.com/2131950/82247839-f69e3c80-98fb-11ea-8bf7-fe67c08ec0a9.png)

The `pos` function cannot be found; the `x` function can be found. The only difference is that `pos` is defined in a file that's a symbolic link and `x` is defined in a regular file. When I looked at the Java stacktrace, the error message was "Could not read from possessive.R because it is not a file."

**Comment by perNyfelt** [comment]
For me it works on 0.9.2726 as well:
```
Renjin 0.9.2726
Copyright (C) 2019 The R Foundation for Statistical Computing
Copyright (C) 2019 BeDataDriven
Falling back to pure JVM BLAS libraries.
> source('link.R')
[1] "Hello there"
```
But there a quite a number of things fixed in the 3.5 beta series. Could you try that in your desktop application?

**Comment by DaveJarvis** [comment]
[PBCAK](https://www.urbandictionary.com/define.php?term=pbcak).
